### PR TITLE
(#599385499) - AAUser on the design system I can see that the TextToggle story is changed

### PR DESCRIPTION
**Pull request by alexwong-crc**
Before submitting my PR to Code Review, I have made sure:

- [ ] I have run the tests using `yarn run test`
- [ ] I have exported any new files in index.js
- [ ] My file supports Formik if will potentially be used in a form
- [ ] The PR name follows the convention `(#<ticket_number>) - <ticket_user_story>`
- [ ] I have incremented the version in `package.json` if I want to publish a new version
- [ ] If my change is breaking I have updated the minor version number(i.e. the middle one) and don't merge until I have the PR on Corazon ready

### CODE REVIEW

File: src/components/atoms/Icon/Icon.tsx [hunk 1]
```diff
@@ -77,7 +77,8 @@ interface IStyledFontAwesomeIcon {
 }
 
 const StyledFontAwesomeIcon = styled(FontAwesomeIcon)<IStyledFontAwesomeIcon>`
-  color: ${props => props.theme[props.color][props.shade].hex};
+  color: ${props =>
+    props.color === "white" ? props.theme.white.hex : props.theme[props.color][props.shade].hex};
```
Comment (elio-stalteri) [inline src/components/atoms/Icon/Icon.tsx:81]: you should include also other colors in this if
```
background: {
      hex: "#F6F6F6",
      RGB: "246,246,246",
    },
    surface: {
      hex: "#FFFFFF",
      RGB: "255,255,255",
      on: "#272727",
    },
    white: {
      hex: "#FFFFFF",
      RGB: "255,255,255",
      on: "#272727",
    },
```

File: src/components/atoms/TextToggle/TextToggle.test.js [hunk 2]
```diff
@@ -18,12 +18,6 @@ describe("TextToggle Component Testing", () => {
     expect(onChange).toHaveBeenCalled()
   })
 
-  it("should pass the width prop to the label container", () => {
```
Comment (elio-stalteri) [inline src/components/atoms/TextToggle/TextToggle.test.js:21]: why is this removed?

File: src/components/atoms/TextToggle/TextToggle.tsx [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+  border-radius: 1.5rem/1.5rem;
+`;
+
+Label.displayName = "Label";
+
+const SliderContainer = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  background-color: ${props => props.theme.black.tint60.hex};
```
Comment (elio-stalteri) [inline src/components/atoms/TextToggle/TextToggle.tsx:26]: why is this fixed color? shouldn't be possible to pass it as a prop?

File: src/components/atoms/TextToggle/TextToggle.tsx [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+  padding: 3px;
+  height: 1.75rem;
+  width: 1.75rem;
+  position: absolute;
+  transition: 0.4s ease;
+  left: 0;
+`;
+
+const Ball = styled.div`
+  background-color: ${props => props.theme.white.hex};
```
Comment (elio-stalteri) [inline src/components/atoms/TextToggle/TextToggle.tsx:73]: shouldn't also this be possible to be passed as a prop??

File: src/components/atoms/TextToggle/TextToggle.tsx [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+Ball.defaultProps = {
+  theme: colourPalette.examplePalette,
+};
+
+Ball.displayName = "Ball";
+
+const OptionText = styled.div`
+  display: flex;
+  flex-direction: column;
+  color: ${props => props.theme.white.hex};
```
Comment (elio-stalteri) [inline src/components/atoms/TextToggle/TextToggle.tsx:89]: shouldn't also this be possible to be passed as a prop??

File: src/components/atoms/TextToggle/TextToggle.tsx [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+  > div {
+    transition: 0.3s;
+    width: 100%;
+  }
+  > div.trueOption {
+    opacity: 0;
+    transform: translateY(50%);
```
Comment (elio-stalteri) [inline src/components/atoms/TextToggle/TextToggle.tsx:96]: nice :)

File: src/components/atoms/TextToggle/TextToggle.tsx [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+      </SliderContainer>
+    </Label>
+  );
+};
+TextToggle.displayName = "TextToggle";
+
+TextToggle.defaultProps = {
+  trueOption: "True",
+  falseOption: "False",
+  width: "4.5rem",
```
Comment (elio-stalteri) [inline src/components/atoms/TextToggle/TextToggle.tsx:164]: why is this needed??

File: src/components/organism/Formik/Formik.stories.js [hunk 8]
```diff
@@ -38,6 +38,7 @@ stories.add(
       }}
       onSubmit={(values, actions) => {
         setTimeout(() => {
+          console.log(values)
```
Comment (elio-stalteri) [inline src/components/organism/Formik/Formik.stories.js:41]: remove logging

File: src/components/atoms/TextToggle/TextToggle.tsx [hunk 9]
```diff
@@ -0,0 +1,168 @@
...
+  isDisabled: false,
```
Comment (elio-stalteri) [inline src/components/atoms/TextToggle/TextToggle.tsx:165]: should this have more default  options?

File: src/components/atoms/TextToggle/TextToggle.test.js [hunk 10]
```diff
@@ -36,3 +30,4 @@ describe("TextToggle Component Testing", () => {
     expect(component.find("Label")).toHaveStyleRule("opacity", "0.4")
   })
 })
+
```
Comment (elio-stalteri) [inline src/components/atoms/TextToggle/TextToggle.test.js:33]: should we add some test for the toggle functionality and onchange callback??

File: src/components/atoms/TextToggle/TextToggle.test.js (hunk 2 above)
Comment (elio-stalteri) [reply src/components/atoms/TextToggle/TextToggle.test.js:21]: cool, we need to remove width from other places also

File: src/components/atoms/TextToggle/TextToggle.tsx (hunk 7 above)
Comment (alexwong-crc) [reply src/components/atoms/TextToggle/TextToggle.tsx:164]: Nice spot!

File: src/components/atoms/TextToggle/TextToggle.stories.js [hunk 11]
```diff
@@ -12,5 +13,21 @@ stories.add("With Text", context => {
...
+  const Confirmed = () => (
+    <div style={{ display: "flex" }}>
+      <Icon name="check" color="white" />
+      <span style={{ marginLeft: "0.5rem" }}>Confirmed</span>
+    </div>
+  )
+
+  return (
+    <TextToggle
+      width="10rem"
```
Comment (elio-stalteri) [inline src/components/atoms/TextToggle/TextToggle.stories.js:26]: this is not needed anymore

File: src/components/atoms/TextToggle/TextToggle.tsx [hunk 12]
```diff
@@ -0,0 +1,167 @@
...
+  isDisabled?: boolean;
+}
+
+const Label = styled.label<ILabel>`
+  position: relative;
+  display: inline-block;
+  height: 1.75rem;
+  overflow: hidden;
+  border-radius: 5px;
+  opacity: ${props => (props.isDisabled ? "0.4" : "1")};
```
Comment (elio-stalteri) [inline src/components/atoms/TextToggle/TextToggle.tsx:15]: can we use object destructuring to get only isDisabled `({isDisabled}) => ...` also below
